Makefile

```diff
@@ -1,7 +1,27 @@
-name := pr/uk-core-functional-tests
+service := pr/uk-core-functional-tests
 repo := europe-west4-docker.pkg.dev/sbat-gcr-develop/sapig-docker-artifact
-tag  := latest
+tests := test
 
 docker:
-	docker build -t ${repo}/securebanking/${name}:${tag} .
-	docker push ${repo}/securebanking/${name}:${tag}
+ifndef tag
+	$(warning no tag supplied; latest assumed)
+	$(eval TAG=latest)
+else
+	$(eval TAG=$(shell echo $(tag) | tr A-Z a-z))
+endif
+ifndef setlatest
+	$(warning no setlatest true|false supplied; false assumed)
+	$(eval setlatest=false)
+endif
+	@if [ "${setlatest}" = "true" ]; then \
+		docker build -t ${repo}/securebanking/${service}:${TAG} -t ${repo}/securebanking/${service}:latest . ; \
+		docker push ${repo}/securebanking/${service} --all-tags; \
+    else \
+   		docker build  -t ${repo}/securebanking/${service}:${TAG} . ; \
+   		docker push ${repo}/securebanking/${service}:${TAG}; \
+   	fi;
+
+test:
+	@echo "Running tests suite '${tests}'"
+	sleep 5s
+	./gradlew cleanTest ${tests}
```

Comment:
Actually, this target won't work without the env vars set.

Either remove it or allow the vars to be supplied, defaulting the values to those used by the CF pipeline to run against dev-core.

Reply:
of course, that is done now